admin-dev/themes/new-theme/package.json

```diff
@@ -73,6 +73,7 @@
     "style-loader": "^0.14.0",
     "tether": "^1.3.2",
     "typeahead.js": "^0.11.1",
+    "uglify-save-license": "^0.4.1",
```

Comment:
is this code removing all licenses from compressed files? This is *something* we don't want isn't it?